apps/designer/app/shared/share-project/share-project2.tsx

```diff
@@ -55,29 +59,54 @@ const Permission = ({
 };
 
 type MenuProps = {
-  permission: Permission;
+  relation: Relation;
```

Comment:
I don't see how you are importing Relation type, weird magic

Reply:
It's declared above no need to import. In case we pass unsupported value it would fail. There could be more Relations than this Component will support.